### PR TITLE
Listwise example & friesian featuretable pad support mask_token and list of strings

### PR DESCRIPTION
**Pull request by cyita**

### CODE REVIEW

File: python/orca/example/learn/tf2/lisewise/listwise_parquet.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+
+    def preprocess(tbl, feature_cols):
+        col_dict = {"collect_list(" + c + ")": c + "s" for c in feature_cols}
+        tbl = tbl.group_by("userid", agg="collect_list")
+        tbl = tbl.rename(col_dict)
+        arr_len = lambda x: len(x)
+        tbl = tbl.apply("ratings", "len", arr_len, dtype="int")
+        min_len = tbl.get_stats("len", "min")["len"]
+        max_len = tbl.get_stats("len", "max")["len"]
```
Comment (jason-dai) [inline python/orca/example/learn/tf2/lisewise/listwise_parquet.py:97]: maybe `tbl.get_stats("len", ["max", "min"])`?

File: python/orca/example/learn/tf2/lisewise/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+conda create -n bigdl python==3.7
+conda activate bigdl
+pip install tensorflow pandas pyarrow pillow numpy
+```
+
+Then download and install latest nightly-build BigDL Orca.
+
+```bash
+pip install --pre --upgrade bigdl-orca[ray]
+pip install --pre --upgrade bigdl-friesian
```
Comment (hkvision) [inline python/orca/example/learn/tf2/lisewise/README.md:19]: Can directly use `bigdl-friesian[train]`?

File: python/friesian/src/bigdl/friesian/feature/table.py [hunk 3]
```diff
@@ -1414,21 +1414,23 @@ def mask(self, mask_cols, seq_len=100):
...
         """
         Add padding on specified column(s).
 
-        :param cols: str or a list of str, the column(s) to be padded with 0s. Each column
+        :param cols: str or a list of str, the column(s) to be padded with mask_tokens. Each column
                should be of list type.
         :param seq_len: int, the length to be padded to for cols. Default is 100.
         :param mask_cols: str or a list of str, the column(s) to be masked with 1s and 0s.
+        :param mask_token: numeric types or str, should be consistent with element's type of cols.
+               default is 0.
```
Comment (hkvision) [inline python/friesian/src/bigdl/friesian/feature/table.py:1426]: default -> Default

File: python/orca/example/learn/tf2/lisewise/README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+We recommend conda to set up your environment. You can install a conda distribution from [here](https://docs.conda.io/projects/conda/en/latest/user-guide/install/)
+if you haven't already.
+
+```bash
+conda create -n bigdl python==3.7
+conda activate bigdl
+pip install tensorflow pandas pyarrow pillow numpy
+```
+
+Then download and install latest nightly-build BigDL Orca.
```
Comment (hkvision) [inline python/orca/example/learn/tf2/lisewise/README.md:15]: Shall we put this example in friesian or orca? Quite strange that in orca example it uses friesian

Comment (jason-dai) [reply python/orca/example/learn/tf2/lisewise/README.md:15]: Yes, it should go to friesian

Comment (cyita) [reply python/orca/example/learn/tf2/lisewise/README.md]: moved to az/friesian https://github.com/analytics-zoo/friesian/pull/239

File: python/orca/example/learn/tf2/lisewise/listwise_parquet.py (hunk 1 above)
Comment (cyita) [reply python/orca/example/learn/tf2/lisewise/listwise_parquet.py]: Fixed in https://github.com/analytics-zoo/friesian/pull/239

File: python/orca/example/learn/tf2/lisewise/README.md (hunk 2 above)
Comment (cyita) [reply python/orca/example/learn/tf2/lisewise/README.md]: Fixed in https://github.com/analytics-zoo/friesian/pull/239

**Comment by cyita** [comment]
Jenkins passed http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-Friesian-PR-Validation/88/

**Pull request merged**